js/main.js

```diff
@@ -202,6 +186,39 @@ function randomNumberForChooseQuote() {
 
 randomNumberForChooseQuote();
 
+function emptyQuote() {
+  mainQuote.textContent = '';
+  miniQuote.textContent = '';
+}
+
+function customQuoteGenerating() {
+  headingPhrase.textContent = 'Custom Quote Generating';
+}
+
+function customQuoteGenerated() {
+  headingPhrase.textContent = 'Custom Quote Generated';
+}
+
+function CustomQuoteGenerating() {
+  headingPhrase.textContent = 'Custom Quote Generating';
+}
```

Comment:
You probably don't need two copies of the same function definition

Reply:
Hey Tim,
When the user clicks on "generate a quote" button, the header text will change to "custom quote generating", and when the user goes through the yes/no and finally selects the "yes" button, the header text will change to "custom quote generated" so I do need both these functions.

https://user-images.githubusercontent.com/55678486/173156074-94ea89b9-5642-43b7-92ed-269e02f8d1c4.mov